flow-report/test/common-test.tsx

```diff
@@ -4,18 +4,20 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {jest} from '@jest/globals';
+import jestMock from 'jest-mock';
 import {act, render} from '@testing-library/preact';
 
+// @ts-ignore: File is not under 'rootDir'
+import {timers} from '../../lighthouse-core/test/test-env/fake-timers.js';
```

Comment:
Ideally this could go in `shared` or something, but that already has a `test` folder and this is the only place we do this so it's probably fine.

Reply:
Moving common tests folder to a common folder will be done sometime later.